content/en/docs/reference/_index.md

```diff
@@ -18,7 +18,7 @@ This section of the Kubernetes documentation contains references.
 
 ## API Reference
 
-* [Kubernetes API Reference {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
+* [API Reference for Kubernetes v{{< param "version" >}}](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
```

Comment:
I think `version` and `latest` variables are equivalent?

Reply:
IMO, `version` is the right param in this context.

Reply:
Interesting. I looked at the previous configurations from 1.14 to 1.20.
The latest = version when the version is not deprecated?

Reply:
Seems like v1.19, v1.18, v1.17 use the `latest` parameter to generated the API reference links.
I've opened PRs to modify the `latest` parameter to the deprecated version in these snapshots.
I agree that "version" makes more sense in this context.

Reply:
I'd expect that `latest` evaluated to the most recent released Kubernetes version, whatever documentation version you're looking at, whereas `version` evaluates to the version number for the documentation you are viewing.

Reply:
Yes, though, how is this parameter used in the context of a deprecated version?